packages/tokens/src/alias/highContrastColor.ts

```diff
@@ -48,7 +48,9 @@ export const generateColorTokens = (): ColorTokens => ({
   colorCompoundBrandForeground1Hover: hcHighlight, // Highlight Global.Color.hcHighlight
   colorCompoundBrandForeground1Pressed: hcHighlight, // Highlight Global.Color.hcHighlight
   colorBrandForeground1: hcCanvasText, // CanvasText Global.Color.hcCanvasText
-  colorBrandForeground2: hcButtonText, // ButtonText Global.Color.hcButtonText
+  colorBrandForeground2: hcCanvasText, // CanvasText Global.Color.hcCanvasText
```

Comment:
The change of `colorBrandForeground2` and `colorBrandBackground2` effectively means switching the colors from black-on-white to white-on-black. This change has already been incorporated on partner side by using an override.

But it breaks `Badge` and `DatePicker`, see screenshot test comment.